doc/quick-start.md

```diff
@@ -6,16 +6,11 @@ jupytext:
     extension: .md
     format_name: myst
     format_version: 0.13
-    jupytext_version: 1.14.4
+    jupytext_version: 1.14.5
 kernelspec:
   display_name: Python 3 (ipykernel)
   language: python
   name: python3
-myst:
```

Comment:
The metadata disappeared for this file.

I recently discovered this problem, jupyter will delete it once we save the file. the solution is to copy it back from the master branch, and add `notebook_metadata_filter: myst`, see the example at the   bottom of this page: https://github.com/ploomber/contributing/blob/main/documentation/notebooks.md

Reply:
@edublancas I copy and paste the whole part from the link you provided. Not sure if it is expected to be done in this way. Is there any specific command you want to add on this file?

Reply:
So I guess Eduardo means:
1. Make [this part](https://github.com/ploomber/jupysql/blob/master/doc/quick-start.md?plain=1#L1-L19) same as what we had in master branch
2. Add `notebook_metadata_filter: myst` inside


Also if `notebook_metadata_filter: myst` is required to add in each `.md` file, we can have another PR to handle other `.md` files

Reply:
(I am not sure if 
```
  cell_metadata_filter: -all
  formats: md:myst
```
should appear in this file